### DISCUSSION TITLE
CI/CD integration command line tool

### DISCUSSION THREAD

**Discussion by rakesh-308** [apache/apisix#8061] [General]
I can see that apisix had rest apis to create/manage routes/services/consumers etc.

Do it also have a command line tool to achieve it so that I can easily integrate it in our ci/cd pipeline. It is tricky to use rest apis there. 

Just for reference Kong has 'deck' as a command line tool on top of its admin apis

**Comment by tzssangglass** [comment]
> Just for reference Kong has 'deck' as a command line tool on top of its admin apis

We are investigating declarative configuration. cc @tokers
